test/models/links_test.rb

```diff
@@ -62,9 +62,8 @@ class LinksTest < ActiveSupport::TestCase
     rubygem = build(:rubygem, linkset: build(:linkset), versions: [version])
     links = rubygem.links(version)
 
-    enumerated = []
-    links.each do |short, value|
-      enumerated << [short, value]
+    enumerated = links.map do |short, value|
```

Comment:
this seems wrong, there is only `each` implemented

https://github.com/rubygems/rubygems.org/blob/master/app/models/links.rb#L41-L47

Reply:
Tests caught it. I'm adding Enumerable to Links to fix it.